lib/ansible/plugins/lookup/hashi_vault.py

```diff
@@ -8,7 +8,8 @@
 DOCUMENTATION = """
   lookup: hashi_vault
   author: Jonathan Davila <jdavila(at)ansible.com>
-  version_added: "2.0"
```

Comment:
This string is meant to show the version of Ansible when the module was added, it shouldn't change